tests/show_test.go

```diff
@@ -106,13 +106,11 @@ func TestShow(t *testing.T) {
 		assert.NoError(t, err)
 		assert.NotContains(t, out, "***")
 		assert.NotContains(t, out, "safecontent=true")
-		assert.NotContains(t, out, "and")
```

Comment:
Had to adjust the integration test, please confirm that this is safe.

Reply:
This is really strange, it should not happen: when safecontent is set to `true`,  the "and" in the twoliner should not be displayed.

Reply:
Did that fail by matching the "and" in this error?
> ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install " + Helpers + ", provide $GOPASS_CLIPBOARD_COPY_CMD and $GOPASS_CLIPBOARD_CLEAR_CMD

Please, revert this and change the content of the twoliner test to have something else that `and\nmore stuff` to see if it works then.

Reply:
Fixed in ecc2333b